.dockerignore

```diff
@@ -0,0 +1,4 @@
+build/
+LICENSE
+README.md
+vendor/
```

Comment:
Doesn't make a difference (yet) as all `COPY`s are explicit, but could add the usual `.git` etc? (So think `.git/`, `.gitignore` and `Jenkinsfile`?)

Reply:
Yes, because it shrinks the size of the package that is sent to the daemon.